frontend/src/routes/Credentials/AddCredentials/AddCredentials.tsx

```diff
@@ -273,7 +273,7 @@ export function AddCredentialPageContent(props: { providerConnection: ProviderCo
 
                     if (getProviderConnectionProviderID(providerConnection) === ProviderID.CRH) {
                         updateProviderConnection((providerConnection) => {
-                            providerConnection.metadata.namespace = multiClusterHubs[0].metadata.namespace
+                            providerConnection.metadata.namespace = multiClusterHubs[0]?.metadata.namespace?.toString()
```

Comment:
I don't understand why the `toString` is needed

Reply:
in case it is undefined, that way it will be "undefined" string

Reply:
Is that desired behavior?

Reply:
updated